controlplane/kubeadm/internal/kubeadm_config_map_test.go

```diff
@@ -47,7 +47,7 @@ kubernetesVersion: v1.16.1
 	delete(kubeadmConfigNoKey.Data, clusterConfigurationKey)
 
 	kubeadmConfigBadData := kconf.DeepCopy()
-	kubeadmConfigBadData.Data[clusterConfigurationKey] = `foobar`
+	kubeadmConfigBadData.Data[clusterConfigurationKey] = `something`
```

Comment:
is this change needed?

Reply:
I was getting a linting error saying `foobar` was used three times and that I should change it to a constant.